### PR TITLE
special case relationships/recovered types

### PR DESCRIPTION
**Pull request by tomjebo**
This relationship type comes from potentially legacy recovered files

### CODE REVIEW

**Comment by tomjebo** [comment]
fixes issue #1205

**Review by mikeebowen — APPROVED** [review]
Looks like a good fix, but the comment should probably be removed.

File: src/DocumentFormat.OpenXml/Packaging/OpenXmlPartContainer.cs [hunk 1]
```diff
@@ -1811,6 +1811,12 @@ internal void LoadReferencedPartsAndRelationships(OpenXmlPackage openXmlPackage,
         {
             foreach (var relationship in relationshipCollection)
             {
+                // http://github.com/
```
[the remark this answers is not in the document]
Comment (tomjebo) [reply src/DocumentFormat.OpenXml/Packaging/OpenXmlPartContainer.cs]: I meant it to point to the issue. I updated the comment.

**Review by mikeebowen — APPROVED** [review]
Approved

**Pull request merged**